test/utils.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Please consider renaming the file to something more meaningful, like `utils/server.ts` perhaps?

Reply:
Applied.